solidity/dashboard/src/contracts.js

```diff
@@ -346,10 +353,19 @@ export const createLPRewardsContract = async (web3, contractName) => {
   return await getContract(web3, artifact, {})
 }
 
-export const createSaddleSwapContract = (web3) => {
+export function createSaddleSwapContract(web3) {
```

Comment:
Just wondering - is there any reason to change `const` to `function` to define function here? 🤔

Reply:
Without going into details(because I don't know how to describe it correctly 😆) is about javascript hoisting. If we use the arrow function the app crashes with `ReferenceError: can't access lexical declaration 'createSaddleTBTCMetaPool' before initialization` because we import functions in `constants.js` from `contracts.js` and `contract.js` file imports from `constants.js`. Not sure if it's an actual solution so we should probably take a look at it in the future.

Reply:
I see, let's keep it that way for now then.